storeclient/src/main/java/org/duracloud/client/PartialContentRetryInputStream.java

```diff
@@ -122,7 +122,12 @@ private void retryFromNextByte() throws IOException {
     }
 
     @Override
-    public int available() throws IOException {
-        return this.currentStream.available();
+    public int available() {
+        return 0;
```

Comment:
Not sure I understand this change. Why would we want available() to always return 0?

Reply:
I'll put in a try catch and return 0 on failure.